<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form {
	background-image: url(./Image/bg.png);         
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="verticalFrame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="startTitle_QLabel">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>印品鸿蒙体（非商用）</family>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	
	color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="back_pushButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>印品鸿蒙体（非商用）</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: rgb(0, 0, 0);
    color: white;
    border: 2px solid #262626;
    border-radius: 14px;
}
QPushButton:hover {
    background-color: rgb(0, 0, 0);
    color: white;
    border: 2px solid  #f48322;
    border-radius: 14px;
}

QPushButton:pressed {
    background-color: #f48322;
    color: black;
    border: 5px solid #f48322;
	border-radius: 14px;
}</string>
        </property>
        <property name="text">
         <string>返回/保存</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="verticalFrame_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>280</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>280</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">.QFrame{
	border: 2px solid #262626;
    border-radius: 14px;
	background-color: #000000;
}
.QWidget{
    background-color: #222222;
    color: white;
    border: 2px solid #262626;
    border-radius: 14px;
}


.QLabel {
	
	color: rgb(255, 255, 255);
}



QComboBox
{
	border-radius:14px;
	background-color:#000000;
    color:rgb(255,255,255);
    border:2px solid #323232;
	padding-left: 10px;
}
QComboBox:disabled
{
	background-color:rgba(50,50,50,200);
    color:rgb(160,160,160);
}
QComboBox:hover
{
	background-color:#000000;
	border:2px solid #a6c100 ;
}
/*点击combox的样式*/
QComboBox:on
{
	border-radius:14px;
	background-color:#000000;
    color:#ffffff;
    border:2px solid #a6c100  ;
}



/*下拉框的样式*/
QComboBox QAbstractItemView 
{
    outline: 0px solid gray;  /*取消选中虚线*/
    border: 0px solid rgb(31,156,220);  
    color: #ffffff;
    background-color: #222222;   
    selection-background-color: #a6c100;   
	border-radius:14px;

}
 /*选中每一项高度*/
QComboBox QAbstractItemView::item
 { 
	
	height: 30px;
	 border-radius: 8px;
    min-height:30px;

 }
/*选中每一项的字体颜色和背景颜色*/
QComboBox QAbstractItemView::item:selected 
{
    color: #000000;
	background-color: #a6c100 ; 
}

QComboBox::drop-down 
{
	border:none;
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QWidget" name="Evasion_setting" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="Evasion_label">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>印品鸿蒙体（非商用）</family>
            </font>
           </property>
           <property name="text">
            <string>闪避</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="Evasion_comBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>30</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="sizeIncrement">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>印品鸿蒙体（非商用）</family>
            </font>
           </property>
           <item>
            <property name="text">
             <string>极限弹闪</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>极限支援</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>极限闪避</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="bounce_Setting" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="bounce_label">
           <property name="font">
            <font>
             <family>印品鸿蒙体（非商用）</family>
            </font>
           </property>
           <property name="text">
            <string>弹反</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="bounce_comboBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>印品鸿蒙体（非商用）</family>
            </font>
           </property>
           <item>
            <property name="text">
             <string>同轴闪避</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>极限双闪</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>极限闪避</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="determineMode_Setting" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="determineMode_label">
           <property name="font">
            <font>
             <family>印品鸿蒙体（非商用）</family>
            </font>
           </property>
           <property name="text">
            <string>判断方式</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="determineMode_comboBox">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="sizeIncrement">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>印品鸿蒙体（非商用）</family>
            </font>
           </property>
           <item>
            <property name="text">
             <string>视觉</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>音频</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="operationMode_setting" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="operationMode_label">
           <property name="font">
            <font>
             <family>印品鸿蒙体（非商用）</family>
            </font>
           </property>
           <property name="text">
            <string>操作方式</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="operationMode_comboBox">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>印品鸿蒙体（非商用）</family>
            </font>
           </property>
           <item>
            <property name="text">
             <string>自动</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>键鼠</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>手柄</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="outTone_Setting" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="outTone_label">
           <property name="font">
            <font>
             <family>印品鸿蒙体（非商用）</family>
            </font>
           </property>
           <property name="text">
            <string>局外提示语音</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="outTone_comboBox">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>印品鸿蒙体（非商用）</family>
            </font>
           </property>
           <item>
            <property name="text">
             <string>开启</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>关闭</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="inTone_Setting" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="inTone_label">
           <property name="font">
            <font>
             <family>印品鸿蒙体（非商用）</family>
            </font>
           </property>
           <property name="text">
            <string>局内提示语音</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="inTone_comboBox">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>印品鸿蒙体（非商用）</family>
            </font>
           </property>
           <item>
            <property name="text">
             <string>开启</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>关闭</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
